conftest.py

```diff
@@ -17,6 +17,7 @@
 import signal
 import time
 import psutil
+import itertools
```

Comment:
Why itertools if no need?

Reply:
Sorry I used it in an earlier intermediate commit. Will remove